sema/import_test.go

```diff
@@ -925,19 +917,27 @@ func TestCheckImportAlias(t *testing.T) {
 			},
 		)
 
-		require.NoError(t, err)
+		errs := RequireCheckerErrors(t, err, 1)
+
+		assert.IsType(t, &sema.DuplicateImportError{}, errs[0])
 	})
 
-	t.Run("invalid duplicate aliases", func(t *testing.T) {
+	t.Run("invalid multiple alias of same element, same import", func(t *testing.T) {
```

Comment:
Here and below: The diff looks like a change, but really is just adding another variant of the subtest above: same import declaration instead of multiple import declarations.